packages/wonder-stuff-core/src/kind-error.js

```diff
@@ -104,13 +109,13 @@ export class KindError extends Error {
         if (kind && /\s/g.test(kind)) {
             throw new Error("kind must not contain whitespace");
         }
-        if (prefix && /\s/g.test(prefix)) {
+        if (prefix != null && /\s/g.test(prefix)) {
```

Comment:
Linting against sketchy null and sketchy number caught a few things that we weren't handling right. So this fixes them up.